phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -3540,11 +3540,18 @@ Collections.<Class<? extends Exception>> singletonList(
                                 if (!ConnectionQueryServicesImpl.this.upgradeRequired.get()) {
                                     if (!isDoNotUpgradePropSet) {
                                         createOtherSystemTables(metaConnection);
+
+                                        // When creating the tables above, DDL statements are
+                                        // used. However, the CFD level properties are not set
+                                        // via DDL commands, hence we are explicitly setting
+                                        // few properties using the Admin API below.
+                                        updateSystemSequenceWithCacheOnWriteProps(metaConnection);
```

Comment:
This call should be pushed into createOtherSystemTables, to improve readability.

Reply:
@stoty Thanks. I have made the changes.

Reply:
@stoty Can you eyeball it ?